addon/components/goal/add-button.js

```diff
@@ -23,16 +23,13 @@ export default Component.extend(ClickOutside, {
       this.toggleProperty('_isBottomSheetShown');
     },
     addContributor(role) {
-      //this.get('model').get('contributors').pushObject(this.get('session.user'));
-      this.get('session.user.goals').pushObject(this.get('model'));
+      this.get('model').get('contributors').pushObject(this.get('session.user'));
```

Comment:
You don't have to chain `get`, instead you can do `this.get('model.contributors')`